cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -588,6 +591,19 @@ func (b *Builder) withKueueLabels(objectMeta *metav1.ObjectMeta) {
 	if len(b.priority) != 0 {
 		objectMeta.Labels[kueueconstants.WorkloadPriorityClassLabel] = b.priority
 	}
+
+	if b.timeLimit != "" {
+		maxExecutionTimeSeconds, err := parser.TimeLimitToSeconds(b.timeLimit)
+		if err != nil {
+			return fmt.Errorf("cannot parse '%s': %w", b.timeLimit, err)
+		}
+
+		if maxExecutionTimeSeconds != nil {
+			objectMeta.Labels[kueueconstants.MaxExecTimeSecondsLabel] = fmt.Sprint(ptr.Deref(maxExecutionTimeSeconds, 0))
```

Comment:
```suggestion
			objectMeta.Labels[kueueconstants.MaxExecTimeSecondsLabel] = fmt.Sprint(*maxExecutionTimeSeconds)
```
since we checked for nil already